assets/js/temp3.js

```diff
@@ -0,0 +1,226 @@
+// Global Variables
+
+var state = {
+  // starting state of questions in DOM - none.
+  items: [],
+  // answers[] is static but will be changed out with every question iteration
+  answers: [
+    '0815',
+    '2B',
+    'BAM128',
+    'Barely'
+  ],
+  // starting value of question when DOM Loads.
+  current_question: 0,
+  // starting value shown as score in DOM.
+  correct: 0,
+  // starting value of counter
+  index: 0,
+  // sets questionHTML to be 'this.'
+  questionHTML: null
+};
+
+var QUESTIONS = [
+    {
+        text: '<:48:x<:65:=<:6C:$=$=$$~<:03:+$~<:ffffffffffffffbd:+$<:ffffffffffffffb1:+$<:57:~$~<:18:x+$~<:03:+$~<:06:x-$x<:0e:x-$=x<:43:x-$',
+        answers: [
+            '0815',
+            '2B',
+            'BAM128',
+            'Barely'
+        ],
```

Comment:
I would personally put all answers in this array all on one line to be less prone to syntax errors.